web/src/realm_playground.ts

```diff
@@ -3,39 +3,58 @@ import * as typeahead from "../shared/src/typeahead";
 
 import {$t} from "./i18n";
 
-const map_language_to_playground_info = new Map();
-const map_pygments_pretty_name_to_aliases = new Map();
+type RealmPlayground = {
+    id: number;
+    name: string;
+    pygments_language: string;
+    url_prefix: string;
+};
 
-export function update_playgrounds(playgrounds_data) {
+const map_language_to_playground_info = new Map<
+    string,
+    Omit<RealmPlayground, "pygments_language">[]
+>();
+const map_pygments_pretty_name_to_aliases = new Map<string, string[]>();
+
+export function update_playgrounds(playgrounds_data: RealmPlayground[]): void {
     map_language_to_playground_info.clear();
 
     for (const data of Object.values(playgrounds_data)) {
```

Comment:
`Object.values` doesn't seem to be necessary anymore given the context. I don't find any caller or legacy API feature level that relies on this implementation detail, so we can iterate over `playgrounds_data` directly here.